src/LuckyBuy.sol

```diff
@@ -45,6 +45,7 @@ contract LuckyBuy is
     uint256 public constant MIN_COMMIT_EXPIRE_TIME = 1 minutes;
     uint256 public constant ONE_PERCENT = 100;
     uint256 public constant BASE_POINTS = 10000;
+    uint256 public constant PAYOUT_FEE_BP = 200; // 2% payout fee
```

Comment:
I think it is worthwhile making this a variable that can tune through a getter/setter